src/ctapipe/image/muon/features.py

```diff
@@ -200,11 +200,13 @@ def ring_size_parameters(
     ring_intensity: float
         Sum of the p.e. inside the integration area of the ring
     intensity_outside_ring: float
-        Sum of the photons outside the ring integration area that passed the cleaning
+        Sum of the p.e. outside the ring integration area that passed the cleaning
     n_pixels_in_ring: int
         Number of pixels inside the ring integration area that passed the cleaning
     mean_intensity_outside_ring: float
-        Mean intensity of the pixels outside the ring, but still close to it
+        Mean intensity of the pixels outside the integration area of the ring,
```

Comment:
Is this the same as l1201 of `ctapipe.containers`? If so, why the docstrings are different?

Reply:
They are the same, but I didn't want to duplicate the exact same text. 
Or should I just duplicate the doc strings from `ctapipe.containers` into the doc strings in the methods from `ctapipe.image.muon.features`?

Reply:
Perhaps this function simply should return `MuonParametersContainer`?
Also, the number of input parameters is very high, consider to wrap them into a structure (dict, container, ...)

Reply:
> Perhaps this function simply should return `MuonParametersContainer`? 

All this module in fact is returning just one `MuonParametersContainer` and I adopted already existing style where each method returning some logically/technically connected part of `MuonParametersContainer`.  Should I reconsider this and return calculated parts of `MuonParametersContainer` from every method?

> Also, the number of input parameters is very high, consider to wrap them into a structure (dict, container, ...)

Good point